easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.2.2-gompi-2025a-fb.eb

```
@@ -0,0 +1,36 @@
+name = 'ScaLAPACK'
+version = '2.2.2'
+versionsuffix = '-fb'
+
+homepage = 'https://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompi', 'version': '2025a'}
+toolchainopts = {'extra_fflags': '-lpthread', 'openmp': True, 'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/Reference-ScaLAPACK/scalapack/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a2f0c9180a210bf7ffe126c9cb81099cf337da1a7120ddb4cbe4894eb7b7d022']
+
+builddependencies = [
+    ('CMake', '3.31.3'),
+]
+
+dependencies = [
+    ('FlexiBLAS', '3.4.5'),
+]
+
+# Config Opts based on AOCL User Guide:
+# https://developer.amd.com/wp-content/resources/AOCL_User%20Guide_2.2.pdf
+
```

```suggestion

preconfigopts = 'export CFLAGS="$CFLAGS -std=gnu89" &&'
```
from https://bugzilla.redhat.com/show_bug.cgi?id=2178710

---

This is still used in the latest Fedora rawhide version, so this looks like the way forward:
https://src.fedoraproject.org/rpms/scalapack/blob/rawhide/f/scalapack.spec#_355